### ISSUE TITLE
NodeVM won't recognize custom classes

### ISSUE THREAD

**Issue by benjsicam** [patriksimek/vm2#9]
```javascript
var vm = require('vm2').NodeVM;

function Data() {
	this.key1 = 'value1';
};

Data.prototype.set = function (field, value) {
	this[field] = value;
};

Data.prototype.get = function (field) {
	return this[field];
};

var sandbox = new vm();

var code = 'module.exports = function (Data) {' +
   'try {' +
      'var data = new Data();' +
      'data.set("key2", "value2");' +
      'return data;' +
   '}' +
   'catch(error) {' +
      'return error;' +
   '}' +
'};'

var fn = sandbox.run(code);
var result = sandbox.call(fn, Data);

console.log(result);
```

The above code returns an error [TypeError: undefined is not a function]

**Comment by rafinskipg** [comment]
it seems that it does not clones the prototype methods. Only the ones declared on the constructor. ¿Can you fix this?

**Comment by patriksimek** [comment]
Fixed in 3.0.
